.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Coda Community Forum
+    url: https://community.coda.io/c/15
+    about: Please ask and answer questions here.
+  - name: Coda Support
+    url: https://help.coda.io/
+    about: Coda's support site.
```

Comment:
Perhaps combine these into one link to the Pack SDK support page:

https://coda.io/packs/build/latest/support/

It covers both as well as additional options.

Reply:
Done